### DISCUSSION TITLE
(use case): chart.js and github-readme-stats

### DISCUSSION THREAD

**Discussion by ghost** [anuraghazra/github-readme-stats#2417] [Q&A] [answered]
Hi everyone.

*idea*
I was thinking of other visual formats for github-readme-stats.

*concept*
![image](https://user-images.githubusercontent.com/111701513/212568201-04922a4e-629b-4b55-9d73-dc32d760ad5b.png)

*source-code*
```javascript
{
  type: 'bar',
  data: {
    labels: ['Stars Earned', 'Commits', 'PRs', 'Issues', 'Contributed'],
    datasets: [{
      label: 'Total Stars Earned',
      data: [53.4]
    }, {
      label: 'Total Commits (2023)',
      data: [164]
    }, {
      label: 'Total PRs',
      data: [581]
    }, {
      label: 'Total Issues',
      data: [134]
    }, {
      label: 'Contributed to (last year)',
      data: [7]
    }]
  }
}
```

*url*
`https://github-readme-stats.vercel.app/api?username=codehangen&show=stars earned,commits, prs, issues, contributed&type=bar`

*question/feedback*
*What do you all think about this idea?*

**Comment by rickstaa** [comment, marked as answer]
@codehangen Thanks for your feature suggestion. If enough people show support for this feature, it will get implemented. You can of course also create a PR for it yourself.👍

**Reply by ghost** [reply]
Hi rickstaa.

> @codehangen Thanks for your feature suggestion. If enough people show support for this feature, it will get implemented. You can of course also create a PR for it yourself.👍

thank you for feedback now.
